test/bridgeAdmin.ts

```diff
@@ -0,0 +1,226 @@
+import { expect } from "chai";
+import { ethers, upgrades } from "hardhat";
+import { SignerWithAddress } from "hardhat-deploy-ethers/signers";
+
+import { AddressOne, convertToUnit } from "../helpers/utils";
+import {
+  AccessControlManager,
+  LZEndpointMock,
+  LZEndpointMock__factory,
+  XVS,
+  XVSBridgeAdmin,
+  XVSProxyOFTDest,
+  XVSProxyOFTDest__factory,
+  XVS__factory,
+} from "../typechain";
+
+describe("Bridge Admin: ", function () {
+  const localChainId = 1;
+  const remoteChainId = 2;
+  const singleTransactionLimit = convertToUnit(10, 18);
+  const maxDailyTransactionLimit = convertToUnit(100, 18);
+
+  let LZEndpointMock: LZEndpointMock__factory,
+    ProxyOFTV2Dest: XVSProxyOFTDest__factory,
+    RemoteTokenFactory: XVS__factory,
+    remoteEndpoint: LZEndpointMock,
+    remoteOFT: XVSProxyOFTDest,
+    bridgeAdmin: XVSBridgeAdmin,
+    remotePath: string,
+    alice: SignerWithAddress,
+    deployer: SignerWithAddress,
+    accessControlManager: AccessControlManager,
+    remoteToken: XVS;
+
+  async function grantPermissions() {
+    let tx = await accessControlManager
+      .connect(deployer)
+      .giveCallPermission(bridgeAdmin.address, "setMaxSingleTransactionLimit(uint16,uint256)", alice.address);
+    await tx.wait();
+
+    tx = await accessControlManager
```

Comment:
The describe block is good for scoping variables and grouping  tests. I wouldn't run code in the describe scope. Its better placed in a hook to be run before all tests or before each test. 

I also think we can use fixtures / deployment scripts so we don't have to copy pasta deployment code between tests.